Lua error when trying to wire an entity with many inputs

When looking with Wire on an entity with many wire inputs you get the error:
```
[ERROR] addons/wire/lua/weapons/gmod_tool/stools/wire_adv.lua:970: bad argument #2 to 'max' (number expected, got nil)
  1. max - [C]:-1
   2. DrawHUD - addons/wire/lua/weapons/gmod_tool/stools/wire_adv.lua:970
    3. unknown - gamemodes/sandbox/entities/weapons/gmod_tool/cl_init.lua:57
```
Example entity made in e2:
```
@inputs  A B C D E F G H I J K L M N O P Q R S T U V W X Y Z
@inputs  A1 B1 C1 D1 E1 F1 G1 H1 I1 J1 K1 L1 M1 N1 O1 P1 Q1 R1 S1 T1 U1 V1 W1 X1 Y1 Z1
@inputs  A2 B2 C2 D2 E2 F2 G2 H2 I2 J2 K2 L2 M2 N2 O2 P2 Q2 R2 S2 T2 U2 V2 W2 X2 Y2 Z2
```

--- a/lua/weapons/gmod_tool/stools/wire_adv.lua
+++ b/lua/weapons/gmod_tool/stools/wire_adv.lua
@@ -892,15 +892,20 @@ elseif CLIENT then
 	end
 	
 	TOOL.CurrentFont = "Trebuchet24"
+	-- Find the largest font that can fit `lines` lines of text into a box `maxsize`
+	-- tall. Set that font as current, and return the size of one line of text.
+	-- If no fonts would fit, then the smallest font possible will be returned.
 	function TOOL:fitFont( lines, maxsize )
 		if not fontheights then
 			getFontSizes()
 		end
+
+		local minFontSize = 14
 		
-		for i=24,14,-2 do
+		for i=24, minFontSize, -2 do
 			local fontname = "Trebuchet" .. i
 			local height = fontheights[fontname]
-			if height * lines <= maxsize then
+			if height * lines <= maxsize or i == minFontSize then
 				self.CurrentFont = fontname
 				surface.SetFont( fontname )
 				local w, _ = surface.GetTextSize( "Selected:" )

